src/Plugin/TfaValidation/TfaTotp.php

```diff
@@ -41,6 +41,8 @@ class TfaTotp extends TfaBasePlugin implements TfaValidationInterface {
    */
   protected $alreadyAccepted;
 
+  protected $context;
```

Comment:
I've learned that this is a new Drupal core thing, we may want to avoid this term or provide documentation on _what_ this is